packages/core/echo/echo-db/src/database.ts

```diff
@@ -253,6 +264,10 @@ export class EchoDatabaseImpl extends Resource implements EchoDatabase {
    * @deprecated
    */
   get objects(): EchoReactiveObject<any>[] {
+    // Initialize all proxies.
+    for (const core of this._automerge.allObjectCores()) {
```

Comment:
this is way to expensive to be done on this getter

Reply:
This getter should not be used at all